helper/resource/testing.go

```diff
@@ -22,6 +22,12 @@ import (
 
 const TestEnvVar = "TF_ACC"
 
+// TestProvider is implemented by scheme.Provider to allow resetting the
+// provider state between tests.
```

Comment:
ah yes, I often forget that schema.Provider technically isn't the only ResourceProvider ;)